lab03/lab03.cpp

```diff
@@ -0,0 +1,83 @@
+﻿
+
+#include <iomanip>
+#include <conio.h>
+
+using namespace std;
+
+
+int main()
+{		
+#pragma region
+	float 
+	tsk2_x= 3, 
+	tsk2_y;
+		
+	double
+		tsk3_t,
+		tsk3_u,
+		tsk3_k = 4,
```

Comment:
It would be better to use preprocessor directive for constant variables  `#define TASK_3_X 5e-5`

Reply:
[Take a look at it here](https://ru.wikibooks.org/wiki/%D0%A1%D0%B8%2B%2B/%D0%9F%D1%80%D0%B5%D0%BF%D1%80%D0%BE%D1%86%D0%B5%D1%81%D1%81%D0%BE%D1%80%D0%BD%D1%8B%D0%B5_%D0%B4%D0%B8%D1%80%D0%B5%D0%BA%D1%82%D0%B8%D0%B2%D1%8B#%D0%94%D0%B8%D1%80%D0%B5%D0%BA%D1%82%D0%B8%D0%B2%D0%B0_define)